apps/web-connect/src/features/hooks/web-buy-keys/useWebBuyKeysOrderTotal.ts

```diff
@@ -51,7 +51,7 @@ export interface UseWebBuyKeysOrderTotalReturn extends UseContractWritesReturn {
     formatItemPricePer: (item: CheckoutTierSummary) => string;
     displayPricesMayVary: boolean;
     nodesAtEachPrice: Array<CheckoutTierSummary> | undefined;
-    chain: Chain | undefined;
+    chain: AppKitNetwork | undefined;
```

Comment:
What is `AppKitNetwork` used for, do we need to have that custm type ?
The next time we appKit changes we will have to update everything all over again.